### ISSUE TITLE
Custom CSS does not work for Mastodon RSS feeds

### ISSUE THREAD

**Issue by mmkthecoolest** [aureliendavid/rsspreview#78]
When viewing a feed for a Mastodon account [(Christian Selig's account as an example)](https://mastodon.social/@christianselig.rss), the custom CSS is not applied. This happens regardless of whether the CSP bypass is enabled.

Custom CSS code to test with:
```CSS
#feedBody {
 max-width: 1100px;
 border: 0px;
 border-radius: 10px;
}
body {
 font-family: sans-serif;
}
video {
 max-width: 100%;
 border-radius: 5px;
}
img {
 border-radius: 5px;
}
.enclosures {
 border: 0px;
 border-radius: 5px;
}
h1 {
 font-size: 200%;
 border-bottom: 0px;
}
@media (prefers-color-scheme: dark){
#feedBody {
 background-color: #222222;
}

#rsspreviewBody {
background-color: black;

}
h2, #feedLastUpdate {
 color: #aaaaaa;
}
a {
 color: #aaaaff;
}

a:active {
 color: #ff5555;
}
.enclosures {
 background-color: #333333;
}
html {
 color: #eeeeee;
 background-color: black;
}
}
```

**Comment by aureliendavid** [comment]
I published a new version on AMO that should fix the issue

please confirm if it's ok for you with v3.20.1

thanks for reporting

**Comment by mmkthecoolest** [comment]
Works when CSP bypass is enabled now, which I am guessing is expected behavior. Feel free to close.

**Comment by mmkthecoolest** [comment]
On second thought now I found another instance where CSP bypass doesn't work. Sorry if that bothers you. https://snowdin.town/users/mjdxp/feed.atom

**Comment by aureliendavid** [comment]
well spotted, this is actually an almost unrelated bug

should be fixed now (3.20.2 on AMO)

thanks for testing

**Comment by mmkthecoolest** [comment]
Seems to be working now. Haven't found any new CSS issues yet.
